todo_app/app.py

```diff
@@ -17,59 +19,77 @@ def create_app():
     SECRET_KEY = os.environ.get("SECRET_KEY", None)
     WRITER_ROLE = os.environ.get("ROLEWRITER_USER", None)  
     redirect_uri=os.environ.get("GITHUB_REDIRECT_URI", None)
-    app.secret_key = SECRET_KEY       
+    app.secret_key = SECRET_KEY     
+    logger = app.logger 
+    logger.setLevel(os.environ.get("LOG_LEVEL", "INFO")) 
+    
+    if os.environ.get("LOGGLY_TOKEN", None) is not None:
```

Comment:
```suggestion
    if os.environ.get("LOGGLY_TOKEN", None):
```

No need for the `is not None`. I think you want to treat empty strings the same way as None